### DISCUSSION TITLE
Checkout Issue with Breeze

### DISCUSSION THREAD

**Discussion by pk-pankaj** [breezefront/community#48] [1. Help]
Hi @vovayatsyuk 

We are not able to checkout at our prod environment with a credit card.

https://mcprod.mishimoto.com/

Please follow the steps to produce the issue:

1. Please add the product: https://mcprod.mishimoto.com/silicone-radiator-hose-kit-dodge-59-cummins-1991-1993.html

2. got to the checkout with zip: 19428 for the US

3. Check and uncheck the "My billing and shipping address are the same" checkbox

4. Enter card info

5. Place an order

Then, getting the error: "The shipping method is missing. Select the shipping method and try again."

We investigate this and found this is happening with the Breeze theme, seems fine with luma

Can you please help us with this, it has been critical for us.

**Comment by vovayatsyuk** [comment]
Breeze is disabled at the checkout page - when you navigate to checkout you are already using Luma stack (requirejs, uiComponents, etc)

I've just tried your checkout and the order was successfully placed. The only thing I noticed is that shipping methods are changing when filling in the address info. Maybe you faced a race condition bug when the shipping method wasn't properly saved.
